Makefile

```diff
@@ -118,11 +119,11 @@ verify-operator-bundle: operator-sdk
 	hack/verify-operator-bundle.sh
 
 operator-bundle:
-	@echo "Building ocs-operator-bundle image"
```

Comment:
Can we keep this info like 
@echo "Building operator-bundle image"

Reply:
We have those in the scripts. I see no need of repeating it here.